app/uk/gov/hmrc/transitmovements/models/DepartureWithoutMessages.scala

```diff
@@ -23,9 +23,10 @@ import java.time.LocalDateTime
 import java.time.OffsetDateTime
 
 case class DepartureWithoutMessages(
-  _id: DepartureId,
+  _id: MovementId,
   enrollmentEORINumber: EORINumber,
   movementEORINumber: EORINumber,
+  movementType: MovementType,
```

Comment:
Remove this from the response model that goes back to the API layer. `MovementType` should be internal to this service only.

Reply:
yes, removed